rest_api/actix_web_1/src/admin/mod.rs

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018-2022 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use splinter::admin::service::AdminService;
+use splinter::rest_api::Resource;
+use splinter::rest_api::RestResourceProvider;
+
+pub struct AdminServiceRestProvider {
+    resources: Vec<Resource>,
+}
+
+impl From<&AdminService> for AdminServiceRestProvider {
+    fn from(_source: &AdminService) -> Self {
+        Self {
+            resources: Vec::new(),
+        }
+    }
+}
+
+impl RestResourceProvider for AdminServiceRestProvider {
```

Comment:
Will all of the `RestResourceProvider` stuff be moving to this crate, eventually?

Reply:
I believe so yes but it will have to be after we move all the other endpoints over